lib/linguist/languages.yml

```diff
@@ -1309,6 +1309,14 @@ DataWeave:
   ace_mode: text
   tm_scope: source.data-weave
   language_id: 974514097
+DenizenScript:
+  type: programming
+  color: "#FBEE96"
+  ace_mode: text
```

Comment:
The language seems superficially similar to yaml, so the following chould be added so the highlighting is also applied to the github code editor:
```suggestion
  ace_mode: yaml
  codemirror_mode: yaml
  codemirror_mime_type: text/x-yaml
```
(e.g.:)
![image](https://user-images.githubusercontent.com/42429413/153702644-edcde9df-cc25-4e0f-b6ba-c9d3ce44b1a1.png)